docs/FeatureManagement/Samples/NoFilters.sample.json

```diff
@@ -0,0 +1,60 @@
+{
+    "feature_management": {
+        "feature_flags": [
+            {
+                "id": "BooleanTrueFeatureFlag",
+                "description": "A simple feature flag, that always returns true.",
```

Comment:
I am ok with saying "a feature flag returns true/false."
But I think it would be more appropriate to say "a feature flag is evaluated as enabled/activated/true".

Reply:
I was trying to avoid the word evaluated as it sort of makes it sound like we look at filters. I swapped it to the description that was in the tests file.

Reply:
The new descriptions look good to me.